scss/_carousel.scss

```diff
@@ -301,6 +301,42 @@
   }
 }
 
+// Boosted mod: specif styles when pause/play button is present
+.carousel-indicators-wrapper {
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  z-index: 2;
+  display: flex;
+  transform: translateX(-50%);
+
+  .carousel-control-play-pause {
+    margin-bottom: $spacer;
+    background: rgba($carousel-indicator-active-bg, .5);
+    border: 0;
+    @include border-radius($spacer, $spacer);
+
+    &.pause {
+      @include button-icon($carousel-control-pause-icon, $carousel-control-pause-button-size, $size: $carousel-control-pause-icon-size, $pseudo: "after");
+    }
+
+    &.play {
+      @include button-icon($carousel-control-play-icon, $carousel-control-pause-button-size, $size: $carousel-control-pause-icon-size, $pseudo: "after");
+
+      &::after {
+        transform: translateX(1px);
+      }
+    }
+  }
+
+  .carousel-indicators {
+    position: initial;
+    margin-left: $carousel-control-pause-indicators-margin-start;
```

Comment:
Should we introduce a selector like `.carousel-control-play-pause + .carousel-indicators` to apply the margin-left ? or is it overkill because people won't use indicators inside indicators wrapper if they don't use play pause button ?

Reply:
After private discussion we decided to keep it as it is.